textacy/datasets/wikipedia.py

```diff
@@ -402,6 +402,8 @@ def records(self, min_len=100, limit=-1, fast=False):
             fast (bool): If True, text is extracted using a faster method but
                 which gives lower quality results. Otherwise, a slower but better
                 method is used to extract article text.
+            keep_section_headings (bool): Whether to include section headings and
```

Comment:
I think I'd prefer to follow the naming convention of the underlying library, and call this `include_headings`. Could you make that change everywhere?

Reply:
what do you think about adding a minimal wikimedia dump file, which just a few pages (could be bogus) as test data so that unit tests (for this and any other changes) could be run against? spot-checking of stuff makes me kind of nervous...

Reply:
Agreed, though that doesn't necessarily have to be in this PR. I have inconsistent test coverage for `textacy`; unfortunately, this particular functionality is not well tested.